_includes/configuration/content_langs.md

```diff
@@ -0,0 +1,23 @@
+## `content_langs`
```

Comment:
This file is not following existing naming conventions... all the current ones use hyphens, not underscores for separators.

- [ ] Fix file names to use hyphens, not underscores